doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -431,7 +453,7 @@ More examples:
 
 #### “Seeing as we have nanobots and power armors…”, “We have teleportation, so it’s not unreasonable to have…”: Irrelevant
 
-If you make this argument, you will not only not make the intended point, since the argument is nonsensical, but you will also damage your credibility with me personally, and I suspect with the other contributors as well. I am absolutely sick of reading this, and I am even more sick of responding to it, so I’ll just refer to this post from now on.
+If you make this argument, you will not only not make the intended point, since the argument is nonsensical, but you will also damage your credibility with me personally, and with many of the other contributors as well. I am absolutely sick of reading this, and I am even more sick of responding to it, so I’ll just refer to this post from now on.
```

Comment:
This point should be moved after other paragraphs of this topic. Presenting objective evidence before subjective one makes the case stronger, as it puts facts before opinions.

Reply:
Ultimately such editorializing is outside of the changes I've made, and not within the scope of what I'm trying to do. If you feel strongly about this I invite you to open a PR to change it, but it's not in scope for this PR. Thanks.